src/hooks/useLongPress.ts

```diff
@@ -108,9 +108,12 @@ const useLongPress = (
   // Web passes React.MouseEvent
   const onContextMenu = useCallback(
     (e: React.MouseEvent | React.PointerEvent) => {
+      // If lock is true (longpress event occurs) we should prevent opening context menu
+      if (lock) {
+        e.preventDefault()
+      }
```

Comment:
Why piggyback on `lock` instead of just checking for right-mouse click? If we're trying to change the behavior only under the condition that the right-mouse button is clicked, it seems like we should test for that directly, unless I'm missing something. Tying it to `lock` tightly couples the right-mouse click behavior to the unrelated lock behavior which was intended to prevent a long press from more than one component at a time.

Reply:
Try rebasing on `main`. The `packageManager` was added and page-lifecycle was fixed in 10b9d7c90faca1aec451bb104386269b4be53c11.